package.json

```diff
@@ -61,6 +61,9 @@
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
       "\\.(css|scss)$": "identity-obj-proxy"
     },
+    "collectCoverageFrom": [
```

Comment:
This fixes the coverage issue where only files with corresponding test files were being used for coverage calculations.